src/AdminConsole/Pages/Organization/Settings.cshtml

```diff
@@ -1,32 +0,0 @@
-﻿@page
-@model SettingsModel
-
-@{
-    ViewData["Title"] = "Organization Settings";
-}
-
-<div id="delete-organization" class="bg-white shadow sm:rounded-lg flex flex-col space-y-4 px-4 py-5 sm:p-6">
```

Comment:
This section gets migrated to `DeleteOrganizationComponent.razor` without additional modifications.